go.mod

```diff
@@ -145,7 +145,7 @@ require (
 	github.com/go-jose/go-jose/v3 v3.0.0 // indirect
 	github.com/go-kit/log v0.2.1 // indirect
 	github.com/go-logfmt/logfmt v0.6.0 // indirect
-	github.com/go-logr/logr v1.2.4 // indirect
+	github.com/go-logr/logr v1.2.4
```

Comment:
can we move all indirect modules to the other require directive?

Reply:
Surprised a `go mod tidy` didn't pick this up.

Reply:
yes, we can revise this later.